membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -118,7 +118,7 @@ object SupporterRatePlanToAttributesMapper {
     ) -> tierThreeTransformer,
     List(
       "8a12831492c341730192dd1c39207038",
```

Comment:
This rate plan ID has now changed (a new one was created when we setup the new product). That's expected right @rupertbates?

Reply:
@david-pepper the new Rate Plan ID: `8a1285e294443da501944b04cb692c9e`

Reply:
Thanks @graham228221, sorry I should have looked up and added these myself! (Using the [product catalog](https://product-catalog.guardianapis.com/product-catalog.json) is a good way to check).

Reply:
Ah yes I didn't think of that, good tip!

Reply:
Done